devicetypes/smartthings/zigbee-multi-button.src/zigbee-multi-button.groovy

```diff
@@ -184,7 +184,7 @@ def initialize() {
 }
 
 private addChildButtons(numberOfButtons) {
-	for(def endpoint : 1..numberOfButtons) {
+	for(def endpoint : 2..numberOfButtons) {
```

Comment:
I don't think this is the right fix. At the very least this will break the other devices that use this DTH. It appears to me that the reason there is an extra button is because this device is using the `generic-4-button-alt` UI Metadata. The only difference between that and `generic-4-button` is the former adds an extra button for the main component. Is there a reason you're not using `generic-4-button` for this device?

Reply:
hi, just tested with generic-4-button, this vid works for the device, I've modified the submit, please check again